stacks-signer/src/cli.rs

```diff
@@ -258,6 +260,23 @@ impl TryFrom<u8> for Vote {
     }
 }
 
+#[derive(Parser, Debug, Clone)]
+/// Arguments for the MonitorSigners command
+pub struct MonitorSignersArgs {
+    /// The Stacks node to connect to
+    #[arg(long)]
+    pub host: String,
+    /// Whether the node is mainnet. Default is false.
+    #[arg(long, default_value = "false")]
+    pub mainnet: bool,
```

Comment:
You can load this from the `host`'s `v2/info` result -- if `{/v2/info}.network_id == 1` then the host is `mainnet`, otherwise, it's testnet. Feel free to ignore this suggestion, though, because it also works to just require this to be set in the CLI.

Reply:
I can do this, but will cause a breaking change in the MockSigning 2.0. As long as we are okay with that can do.

Reply:
This is fine to leave as is then.